python/tank/deploy/tank_commands/install.py

```diff
@@ -36,9 +36,16 @@ def __init__(self):
         self.parameters["environment"] = { "description": "Name of environment to install into.",
                                            "default": None,
                                            "type": "str" }
+        
         self.parameters["engine_instance"] = { "description": "Name of the engine instance to install into.",
                                                "default": None,
                                                "type": "str" }
+        
+        self.parameters["preserve_yaml"] = { "description": ("Enable alternative yaml parser that better preserves "
```

Comment:
It would be cool if there was a way to turn this to true by default. Env var or otherwise.